packages/material-ui/src/ButtonBase/Ripple.js

```diff
@@ -1,13 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
+import { makeStyles } from '@material-ui/styles';
 import { Transition } from 'react-transition-group';
 
+const useStyles = makeStyles(
+  {
+    root: {
```

Comment:
It would be more appropriate here:

```suggestion
    root: props => ({
```